### ISSUE TITLE
Improve the readability for the output from kpt live apply/destroy/status

### ISSUE THREAD

**Issue by mortent** [kptdev/kpt#2251]
The `events` output can be hard to read for a couple of reasons:
* The output corresponding to the different phases can be confusing, in particular when a task doesn't do anything (like the output for prune when no resources have been pruned):
```
service/wordpress created (preview)
service/wordpress-mysql created (preview)
deployment.apps/wordpress created (preview)
deployment.apps/wordpress-mysql created (preview)
persistentvolumeclaim/mysql-pv-claim created (preview)
persistentvolumeclaim/wp-pv-claim created (preview)
6 resource(s) applied. 6 created, 0 unchanged, 0 configured, 0 failed (preview)
0 resource(s) pruned, 0 skipped, 0 failed (preview)
```
* The distinction between create, unchanged, configured and failed are not intuitive, although it does follow the terminology from kubectl.
* Formatting of resource id, status and message for each line can be hard to parse.

**Comment by frankfarzan** [comment]
As part of this or separate issue, we should also look at failure cases. For example:

```
installing inventory ResourceGroup CRD.
service/wordpress unchanged
service/wordpress-mysql configured
deployment.apps/wordpress unchanged
deployment.apps/wordpress-mysql configured
persistentvolumeclaim/mysql-pv-claim configured
persistentvolumeclaim/wp-pv-claim apply failed: error when applying patch:
{"metadata":{"annotations":{"kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"v1\",\"kind\":\"PersistentVolumeClaim\",\"metadata\":{\"annotations\":{\"config.k8s.io/owning-inventory\":\"06ca0268f3ccda82bfd44bec273530a4f72fa9a7-1623278424452000000\"},\"labels\":{\"app\":\"wordpress\"},\"name\":\"wp-pv-claim\",\"namespace\":\"default\"},\"spec\":{\"accessModes\":[\"ReadWriteOnce\"],\"resources\":{\"requests\":{\"storage\":\"30Gi\"}}}}\n"}},"spec":{"resources":{"requests":{"storage":"30Gi"}}}}
to:
Resource: "/v1, Resource=persistentvolumeclaims", GroupVersionKind: "/v1, Kind=PersistentVolumeClaim"
Name: "wp-pv-claim", Namespace: "default"
for: "deployment/volume.yaml": persistentvolumeclaims "wp-pv-claim" is forbidden: only dynamically provisioned pvc can be resized and the storageclass that provisions the pvc must support resize
6 resource(s) applied. 0 created, 2 unchanged, 3 configured, 1 failed
0 resource(s) pruned, 0 skipped, 0 failed
error: 1 resources failed
```

- Consistency with other commands on how to refer to resources
- Blank lines for improved readability
- Currently there's a raw JSON rendering of resource

**Comment by seans3** [comment]
I think the "configured" output is awkward. Probably should be "updated".

Otherwise, I think there is a lot of value when the output is similar to kubectl. All of our customers are familar with this ubiquitous tool. kubectl is the only tool that ships with every Kubernetes release, and even helm and terraform users are familiar with it. I do not believe we should change this output until it is clear from our customers that there is a problem.

**Comment by liamfallon** [comment]
This is not a priority at the moment.
